workshops/templates/base_without_navbar.html

```diff
@@ -7,12 +7,12 @@
     <link rel="stylesheet" href="{% static 'css/amy.css' %}" />
     <link rel="stylesheet" href="{% static 'css/bootstrap-datepicker3.min.css' %}" />
     <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />
+    {% block extrastyle %}{% endblock %}
```

Comment:
Hey @shapiromatron,

Sorry, I wasn't clear.

You're using `block extrastyle` to add one tag call: `include_ui_theme` (for example here: https://github.com/swcarpentry/amy/pull/248/files#diff-780224fa654fedfe332f47ca01cca44fR7). This tag, `include_ui_theme`, in an [inclusion tag](https://docs.djangoproject.com/en/1.8/howto/custom-template-tags/#inclusion-tags). It simply includes `selectable/jquery-css.html` template.

That template is here: https://github.com/mlavin/django-selectable/blob/master/selectable/templates/selectable/jquery-css.html

It breaks some of our tests, because it doesn't follow XHTML syntax, and we rely upon XML parser in the tests.

So we want to override that template. We have to put it in our application (overriding it in `django-selectable` sources is not a good option).

To override that template, add `workshops/templates/selectable/jquery-css.html` with following content:
```html
<link rel="stylesheet" href="//ajax.googleapis.com/ajax/libs/jqueryui/{{ version }}/themes/{{ theme }}/jquery-ui.css" type="text/css" />
```

I hope I'm explained well this time, but ask if something isn't clear. Cheers!